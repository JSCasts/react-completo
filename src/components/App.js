import React from 'react';
import { Router, Route, browserHistory } from 'react-router';

import Shots from './shots/Shots';

const App = () => (
  <Router history={browserHistory}>
    <Route path="/" component={Shots} />
  </Router>
);

export default App;
