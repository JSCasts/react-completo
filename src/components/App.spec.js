import React from 'react';
import renderer from 'react-test-renderer';
import { Router } from 'react-router';

import App from './App';

test('App', () => {
  const component = renderer.create(
    <App />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
