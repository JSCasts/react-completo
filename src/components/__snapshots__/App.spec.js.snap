exports[`test App 1`] = `
<div>
  <header
    className={undefined}>
    <a
      href="#/"
      onClick={[Function]}
      style={Object {}}>
      <img
        alt="Logo"
        src="test-file-stub" />
    </a>
  </header>
  <main
    className={undefined}>
    <div />
  </main>
</div>
`;
